marketplace-service/src/main/java/ch/akros/marketplace/service/service/TopicService.java

```diff
@@ -108,9 +108,10 @@ private FieldOptionResponseDTO toFieldOptionsResponseDTO(FieldOption fieldOption
         return result;
     }
 
-    public void saveTopic(String json, MultipartFile[] files, MultipartFile thumbnail) throws IOException {
+    public void saveTopic(String json, MultipartFile[] files, MultipartFile thumbnail, String userId) throws IOException {
         TopicSaveRequestDTO topicSaveRequestDTO = deserializeStringToTopicSaveRequestDTO(json);
         Topic topic = new Topic();
+        topic.setUserId(userId);
```

Comment:
Have you done unit tests for the saveTopic method with new UserId?

Reply:
I have added it to the current tests and it seems that they still work :)